pkg/didcomm/protocol/didexchange/service_test.go

```diff
@@ -10,10 +10,12 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
```

Comment:
should be grouped with the other Aries imports.

Reply:
done

Reply:
done